### PR TITLE
Update ci and update-snapshots for simple flavor selections.

### PR DESCRIPTION
**Pull request by dmceachernmsft**
# What

Add jobs to ci.yml and update-snapshots.yml to select the matrix before each run from a json string.
# Why

When creating a release branch this will allow the dev in charge of release to just edit the json string to disable flavors instead of having to remove each flavor in the matrix definition.

https://skype.visualstudio.com/SPOOL/_workitems/edit/2806342
# How Tested

Ran on GH to ensure actions work.

### CODE REVIEW

File: .github/workflows/run-td-build.yml [hunk 1]
```diff
@@ -7,6 +7,9 @@ on:
     branches:
       - main
       - release/**
+      - prerelease-beta/**
```
Comment (JamesBurnside) [inline .github/workflows/run-td-build.yml:10]: Same here, this looks unrelated to PR description:
image

File: .github/workflows/create-prerelease-branch.yml [hunk 2]
```diff
@@ -0,0 +1,117 @@
+# Bump package version numbers and bundle changelogs into changelog file
```
[the remark this answers is not in the document]
Comment (JamesBurnside) [reply .github/workflows/create-prerelease-branch.yml:1]: Oh nvr mnd this because you need to merge in main again

**Review by JamesBurnside — APPROVED** [review]

**Review by prprabhu-ms — APPROVED** [review]
